README.md

```diff
@@ -18,6 +18,7 @@ Below is current list of languages and capabilities implemented
 |🇪🇸 🇲🇽 Spanish|✅ Yes|✅ Yes|❌ No|❌ No|
 |🇹🇷 Turkish|✅ Yes|❌ No|❌ No|❌ No|
 |🇺🇦 Ukrainian|✅ Yes|✅ Yes|✅ Yes|❌ No|
+|🇯🇵 Japanese|✅ Yes|❌ No|❌ No|❌ No|
```

Comment:
Well, maybe adding this line before the Japanese update is a bad idea... Hold off on this commit until later?